src/store/modules/job/actions.ts

```diff
@@ -470,7 +470,7 @@ const actions: ActionTree<JobState, RootState> = {
             'systemJobEnumId_op': 'equals'
           }
         })
-        const fetchJobsResponse = jobs.find((job: any) => job?.jobId === payload.jobId);
+        const fetchJobsResponse = jobs.find((job: any) => job?.data.docs[0].jobId === payload.jobId);
```

Comment:
Improve fetchJobsResponse variable name and as we have only one response

Reply:
Improved the variable name to `jobResponse`. 

As we are fetching both draft and pending jobs from the `fetchJobs` action, a find is required to search. 

Also, if we get an empty response from `fetchJobs` actions and find is ran, the `catch` block is executed.